boefjes/boefjes/plugins/kat_dns/main.py

```diff
@@ -21,40 +21,39 @@ def run(boefje_meta: BoefjeMeta) -> List[Tuple[set, Union[bytes, str]]]:
     hostname = boefje_meta.arguments["input"]["name"]
 
     requested_dns_name = dns.name.from_text(hostname)
-    zone_soa_record = get_parent_zone_soa(requested_dns_name)
+    resolver = dns.resolver.Resolver()
+    resolver.nameservers = [str(settings.remote_ns)]
+
+    zone_soa_record = get_parent_zone_soa(resolver, requested_dns_name)
 
     answers = [
         zone_soa_record,
     ]
 
-    dns_record_types = ["A", "AAAA", "TXT", "MX", "NS", "CNAME", "DNAME"]
+    dns_record_types = ["A", "AAAA", "CAA", "CERT", "RP", "SRV", "TXT", "MX", "NS", "CNAME", "DNAME"]
```

Comment:
I don't think the CERT and RP records are much in use, do we really want to fetch those? And if we fetch them shouldn't we also normalize them? Just fetching them to include them in the raw file doesn't feel very useful.

Reply:
We don't need to normalize everything that we have, but those seemed to provide some additional useful information which I plan on including in the model at a later date.